nuxeo-runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/stream/StreamServiceImpl.java

```diff
@@ -225,4 +239,200 @@ public void beforeStop(ComponentManager mgr, boolean isStandby) {
             Framework.getRuntime().getComponentManager().removeListener(this);
         }
     }
+
+    // ---------------------------------------------------------------
+    // Debug tools for internal use
+
+    /**
+     * Display up to {@code limit} record from a Log.
+     */
+    public void cat(String logConfig, String name, int limit) throws Exception {
```

Comment:
Please don't use `throws Exception` in any class of `src/main/java`, even though these are debug methods the code checkers will not like it.

Reply:
yes the code is taken from a unit test and need to be adapted, I will merge 1.0.0 upgrade  without this commit